openmdao/core/indepvarcomp.py

```diff
@@ -48,6 +48,13 @@ def __init__(self, name=None, val=1.0, **kwargs):
         self._indep_external = []
         self._indep_external_discrete = []
 
+        if 'tags' not in kwargs:
+            kwargs['tags'] = []
+        else:
+            if isinstance(kwargs['tags'], str):
+                kwargs['tags'] = [kwargs['tags'], ]
+        kwargs['tags'].append('indep_var_comp')  # Tag all indep var comps this way
```

Comment:
since we are tagging a var, not a comp, should this just be `indep_var` or even just `indep`?  @JustinSGray

Reply:
I second this suggestion.

Reply:
yeah, should be `indep_var` for sure. We'll have to add the same tag to some dymos stuff ....  of course, for that one I think I need to tag only the first index of an array variable... welp, thats another wrinkle for a future story